lunches.py

```diff
@@ -552,6 +552,26 @@ def assen(dom):
     yield from menicka_parser(dom)
 
 
+@restaurant("Bistro Paulus", "https://www.bistro-paulus.cz/poledni-menu/", Location.Olomouc)
+def paulus(dom):
+    current_day = datetime.datetime.now().strftime("%-d.%-m.%Y")
+    for day_dom in dom.css(".section-day"):
+        day = "".join(day_dom.css_first("h3").text(strip=True).split()[1:])
+        if current_day not in day:
+            continue
+        soup_table = day_dom.css("table")[0].css("span")
+        lunch_table = day_dom.css("table")[1].css("span") + day_dom.css("table")[2].css("span")
```

Comment:
I would move this variable just before the right `for` cycle - to show "lifetime" or when the variable is really needed